Gameplay/misc/CameraPosChange.h

```diff
@@ -22,15 +22,16 @@ namespace Hachiko
 			void OnStart() override;
 			void OnUpdate() override;
 		private:
-			GameObject* _player;
-			PlayerCamera* _camera;
+			GameObject* _player{};
+			PlayerCamera* _camera{};
 			bool _is_inside = false;
 			SERIALIZE_FIELD(GameObject*, _objective);
 			SERIALIZE_FIELD(bool, _do_look_ahead);
 			SERIALIZE_FIELD(float, _speed);
 			SERIALIZE_FIELD(float, _duration);
 			SERIALIZE_FIELD(float3, _relative_position);
 			SERIALIZE_FIELD(float3, _rotation);
+			SERIALIZE_FIELD(float4, _rotationf4);
```

Comment:
I believe this was forgotten as it's never used. Let's delete this.

Reply:
oh yeah, that was test code, thanks for the catch